packages/main/src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Form } from './index';
+import React from 'react';
+import notes from './Form.md';
+import { FormItem } from './FormItem';
+import { FormGroup } from './FormGroup';
+import { CheckBox, Input, InputType, Option, Select } from '../..';
```

Comment:
This is still not correct. Please import each of those components separately. e.g.
`import { CheckBox } from '@ui5/webcomponents-react/lib/CheckBox';`